roles/rollback/tasks/main.yml

```diff
@@ -1,22 +1,31 @@
 ---
+- name: Get real path of current symlinked release
+  command: "readlink {{ project_current_path }}"
+  args:
+    chdir: "{{ project_root }}"
+  register: current_release_readlink_result
+
+- name: Clean up old and failed releases
```

Comment:
Do we need this task then? They get cleaned on deploys

Reply:
Still needed because deploy cleanup happens at https://github.com/roots/trellis/blob/ac87cd5af42ea8194499606119b610db265ec501/roles/deploy/tasks/finalize.yml#L8-L14

If deploy fails before that task, there will be invalid release on the server.